packages/components/organisms/f-checkout/src/components/AgeVerification.vue

```diff
@@ -116,7 +116,8 @@ export default {
 
         months () {
             return new Array(12).fill(0).map((_, i) => {
-                const date = new Date(`${i + 1}/1`);
+                const year = new Date().getFullYear();
```

Comment:
Do you need this constant here?

Reply:
I do not. Have removed and got this value in the instantiation of the date below.